partitioned-table.md

```diff
@@ -841,6 +926,20 @@ alter table members optimize partition p0;
 ERROR 8200 (HY000): Unsupported optimize partition
 ```
 
+### Key partition management
+
+Currently, Key partitioning only supports the `ALTER TABLE ... TRUNCATE PARTITION` partition management statement.
+
+If you execute a Key partition management statement that is not yet supported, TiDB will return an error.
```

Comment:
Please limit the use of "will" and try to use the present tense.
```suggestion
If you execute a Key partition management statement that is not yet supported, TiDB returns an error.
```

Reply:
Thanks for the comments. The other places are updated in https://github.com/pingcap/docs/pull/12963/commits/00def2a14f5a99297b89e783528c880f01e40c21.